src/Sylius/Bundle/ReviewBundle/composer.json

```diff
@@ -0,0 +1,56 @@
+{
+    "name": "sylius/review-bundle",
+    "type": "symfony-bundle",
+    "description": "Review system for Symfony2 ecommerce applications.",
+    "keywords": ["review", "shop", "webshop", "ecommerce", "sylius"],
+    "homepage": "http://sylius.org",
+    "license": "MIT",
+    "authors": [
+        {
+            "name":     "Paweł Jędrzejewski",
+            "homepage": "http://pjedrzejewski.com"
+        },
+        {
+            "name":     "Mateusz Zalewski",
+            "email":    "mateusz.zalewski@lakion.com"
+        },
+        {
+            "name":     "Daniel Richter",
+            "email":    "nexyz9@gmail.com"
+        },
+        {
+            "name":     "Sylius project",
+            "homepage": "http://sylius.org"
+        },
+        {
+            "name":     "Community contributions",
+            "homepage": "http://github.com/Sylius/Sylius/contributors"
+        }
+    ],
+    "require": {
+        "php":                             ">=5.5.9",
+        "symfony/framework-bundle":        "^2.7",
+        "stof/doctrine-extensions-bundle": "~1.1",
+        "sylius/resource-bundle":          "0.16.*",
+        "sylius/review":                   "0.16.*"
```

Comment:
We no longer align these, please update all composer.jsons according to other bundles.